### ISSUE TITLE
Output to file

### ISSUE THREAD

**Issue by dotellie** [ai/size-limit#94]
It would be nice if there were a command parameter one could use to save the output of size-limit to a machine readable file. My primary reason for wanting this would be to utilize [GitLab's metrics report feature](https://docs.gitlab.com/ee/ci/metrics_reports.html) but I'm sure there are many other use cases that can come from doing something like this.

**Comment by ai** [comment]
Good idea. I will ask my friends to implement it.

**Comment by ai** [comment]
What do you think about this output?

```bash
$ npx size-limit --json
[
  {
    name: "index.js",
    passed: true,
    size: 1024,
    loading: 0.05,
    running: 0.1
  }
]
```

**Comment by dotellie** [comment]
Yep, that could totally work! Seems very flexible too 😄

**Comment by ai** [comment]
I created a task (it is our Russian board for open source tasks) and will promote it on Twitter.
https://cultofmartians.com/tasks/size-limit-json.html

Hope, people will fix it on this weekend on next one.

Feel free to send PR if you want it earlier.

**Comment by ai** [comment]
Fix https://github.com/ai/size-limit/pull/97 by @vostrik and released in 1.2.0 version.
